docs/tutorial/index.md

```diff
@@ -0,0 +1,12 @@
+# Tutorial: Extending Platform UI with new navigation
```

Comment:
I have a proposition to do a content index.md page: it's more interesting than having a bunch of links to other files.
Should we put this "navigation menu" on the bottom of every file?